process/tech-due-diligence-guidelines.md

```diff
@@ -0,0 +1,103 @@
+
+#### Team
+
+* What is the general quality of informal communication around the project (slack, github issues, PR reviews, technical blog posts, etc)?
+* Would you hire the team? (this is of course a purely hypothetical question).  If not, why?
+* How much time do they commit to the project?
+* How big is the team? Who funds them? Why? How much? For how long?
+* Who are the clear leaders?  Are there any areas lacking clear leadership?  Testing? Release? Documentation?  These roles sometimes go unfilled.
+* Besides the core team, how active is the surrounding community?  Bug reports?  Assistance to newcomers? Blog posts etc.
+* Do they make it easy to contribute to the project? If not, what are the main obstacles?
+* Are there any especially difficult personalities to deal with?  How is this done?  Is it a problem?
+
+#### Customers
+
+* Who uses the product?  Get a few in-depth references from 2-4 of them who actually know and understand it.
+* What do real users consider to be it's strengths and weaknesses?  Any concrete examples of these?
+* Perception vs Reality:  Is there lots of buzz, but the software is flaky/untested/unused?  Does it have a bad reputation for some flaw that has already been addressed?
+
+#### Product
+
+* Where does it fit in the market and technical ecosystem?
+* Is it growing or shrinking in that space?  Is that space growing or shrinking?
+* How necessary is it? What do people who don't have this product do?  Why exactly is that not adequate, and in what situations?
+* What are the most important holes? No HA? No flow control? Inadequate integration points?
+* Clearly compare and contrast with peers in this space.  A summary matrix often helps.
+   Beware of comparisons that are too superficial to be useful, or might have been manipulated so as to favor some products over others.
+   Most balanced comparisons will include both strengths and weaknesses, require significant detailed research, and usually there is no hands-down winner.
+   Be suspicious if there appears to be one.
+
+#### Other advice
+
+* Bring in other people (e.g. from your company) who might be more familiar with a
+  particular area than you are, to assist where needed.  Even if you know the area,
+  additional perspectives from experts are usually valuable.
+* Conduct as much of the investigation in public as is practical.  For example, favor explicit comments on the
+  submission PR over private emails, phone calls etc.  By all means conduct whatever communication might be
+  necessary to do a thorough job, but always try to summarize these discussions in the PR so that others can follow along.
+* Fact-check where necessary.  If an answer you get to a question doesn't smell right, check the underlying data, or get a second/third... opinion.
+
```

Comment:
Worth mentioning conflict of interest and abstaining from tech DD if such a conflict may exist?

Reply:
Typically the sponsor and champions do have a vested interest in the project. If it's not obvious, disclosing it would be helpful.

Reply:
Thanks @lachie83.  I added some verbage around disclosing potential conflicts of interest, and avoiding as necessary.